### PR TITLE
move shared components to quill component library

### PR DESCRIPTION
**Pull request by emilia-friedberg**
Addresses issue #

**Changes proposed in this pull request:**
- move input, dropdown input components and scss to component library
- move card and button scss to component library
- create card component in component library and refactor code to use it
- install and adjust new components in the lms 

**If this is a visual change please attach screencaps of the new branch, making sure to censor user data**

**Has this branch been QA'd on staging?** no

**Have you updated the docs?** no

**Have the tests been updated?** no

**Reviewer:** @anathomical

### CODE REVIEW

**Comment by emilia-friedberg** [comment]
@anathomical do you think it's worth suspending the question of what to do about the dist files for now and coming back to it later? asking because I have something I want to add to the component library but I don't want the versions to get overwritten

**Comment by anathomical** [comment]
@emilia-friedberg Yeah, think that's a good call.  No point in dragging this on forever when we don't know when we'll have time to fix it.  I'll go ahead an approve this so we can get it done and put in an issue to clean it up later.

File: packages/quill-component-library/src/styles/input.scss [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    -moz-transition: all 0.2s ease-in;
+    -o-transition: all 0.2s ease-in;
+    transition: all 0.2s ease-in;
+  }
+  .dropdown {
+    margin-bottom: 8px;
+    &:focus {
+      outline: none;
+    }
+    .dropdown__dropdown-indicator {
```
[the remark this answers is not in the document]
Comment (emilia-friedberg) [reply packages/quill-component-library/src/styles/input.scss]: it's the library we use, react-select: you can't name the classes it uses directly, but you can pass the component a prefix which it'll use to name them, so in this case the prefix is "dropdown."

[the remark this answers is not in the document]
Comment (emilia-friedberg) [reply packages/quill-component-library/src/styles/input.scss]: otherwise it's a string of numbers that I don't think are necessarily consistent

**Pull request merged**